### DISCUSSION TITLE
What's the variable name that let you use "Tab length"?

### DISCUSSION THREAD

**Discussion by rafa4mat** [WinEdt-Team/WinEdt-Support#91] [1.1 General Help and Discussion]
In the .edt files to insert a Envitoment, you can see in many lines "GotoCol(%!0+2)" to indent any lines in 2 spaces.

I´m thinking about indent lines in the "Tab length" spaces setted  at Option Interface / Font schemes / Tabs / Tab length=4

using GotoCol(%!0+ TAB LENGTH)

or with a "tab" 

Is this possible?, how? in function of the tab setted in WinEdt.

**Comment by Alex-Simonic** [comment]
This is currently not possible. I never needed to get the tab length inside macros, so I did not introduce such variables. For what it is worth the next version now has a new variable %!u that returns the length of a tab (as specified in Font Schemes)...

**Reply by rafa4mat** [reply]
No problem for my needs.

I can use

    GotoCol(%!0);Tab;Tab;
    Ins("%!7");

Thank you, Alex.
